pulsar-io/mongo/src/main/java/org/apache/pulsar/io/mongodb/MongoSink.java

```diff
@@ -68,7 +68,7 @@ public class MongoSink implements Sink<byte[]> {
 
     private MongoCollection<Document> collection;
 
-    private List<Record<byte[]>> incomingList;
+    private List<Record<byte[]>> incomingList = Lists.newArrayList();
```

Comment:
```suggestion
    private List<Record<byte[]>> incomingList = new ArrayList<>();
```
It's recommended to use `ArrayList` instead of `Lists.newArrayList()` in guava annotation.

Reply:
@Shoothzj Done.